R/curl.R

```diff
@@ -161,6 +167,27 @@ curl_normalize <- function(cmd) {
     data = data
   )
 }
+
+curl_prepare_headers <- function(headers, simplify_headers) {
+  if (simplify_headers) {
+    header_names <- tolower(names(headers))
+    to_drop <- startsWith(header_names, "sec-fetch") |
+      startsWith(header_names, "sec-ch-ua") |
+      header_names %in% c("user-agent", "referer", "pragma", "connection")
```

Comment:
I think it's worth keeping `user-agent` since that's sometimes important to get the same results